lib/face_control/checkers/comments.rb

```diff
@@ -0,0 +1,29 @@
+require 'face_control/comment'
+
+module FaceControl
+  module Checkers
+    class Comments
+
+      def command(filenames)
+        "grep -inEH '(todo|fixme)' #{filenames}"
+      end
+
+      def parse(command_output)
+        comments = []
+        unless command_output.empty?
+          command_output.each_line do |line|
+            file, line_num, description = line.split(":", 3)
+            comments.push(
+              Comment.new(
+                file: file,
+                line: line_num,
+                text: "File includes the annotation keyword: #{description}"
```

Comment:
How about a more actionable comment? Something like "Do not bury this task in code. Do it now or create a JIRA issue."?

Also, there is no need to include the description into the comment because the entire line will be visible right above the posted comment in the diff.